feat-content.html

```diff
@@ -1,23 +1,35 @@
 <div class="feat-content-container">
-    <div class="feat-content-card">
-        <a class="cardClick" target="_blank" rel="noopener noreferrer"
+    <div class="feat-content-card p-4">
+        <a class="cardClick no-underline" target="_blank" rel="noopener noreferrer"
            href="organizations/HumanPangenome">
-            <h4 class="feat-content-title">Workflows for building and using the Human Pangenome Reference</h4>
-            <p class="feat-card-description">This organization continually adds workflows used to build the Human Pangenome Reference. Learn how to use the latest pangenome reference with the tutorial linked on their page.</p>
+            <div>Featured Organization</div>
+            <h5 class="feat-content-title weight-bold pb-2">Human Pangenome Reference Consortium</h5>
+            <div class="feat-card-description p-1">
+                <p>This organization continually adds workflows used to build the Human Pangenome Reference. Learn how to use the latest pangenome reference with the tutorial linked on their page.</p>
+                <div class="feat-card-btn-container"><a class="mat-raised-button small-btn-structure mat-accent no-underline">View</a></div>  
```

Comment:
Should we substitute a `button` element for the inner `a`?  (I don't know the answer.)

Reply:
> You're not supposed to nest an anchor within an anchor: https://stackoverflow.com/questions/13052598/creating-anchor-tag-inside-anchor-tag.
> 
> I noticed it when I used an online HTML validator; we should at some point probably add an HTML validator to the build process.

Thanks for pointing that out! and good idea

> Should we substitute a `button` element for the inner `a`? (I don't know the answer.)

This is true and what I originally tried to do, but the button element doesn't get rendered through `innerHTML`, because of [sanitization](https://netbasal.com/angular-2-security-the-domsanitizer-service-2202c83bd90). A [workaround](https://stackoverflow.com/questions/56287237/angular-innerhtml-property-not-working-while-try-to-render-button-control) is to bypass sanitization with the  `DomSanitizer` `bypassSecurityTrustHtml()` method then the button will render (I played with it yesterday), but in doing so we are trusting that the data received when getting `feat-content.html` is safe, which is why I decided to workaround and style a `<a>` to look like a `<button>`...

Though if we believe this is a safe method we could use it and switch to a button, but I'm thinking I can just refactor the layout and have the `<a>`s not be nested

Reply:
Pushed some changes, moved the second anchor (View "button") outside like so, so the clickable areas are as follows:

![Group 6](https://user-images.githubusercontent.com/97123241/228965992-1d3fd9a4-c133-4caa-be2f-270ecfe7027c.png)

Downside is the link needs to be declared twice. But a good working solution without bypassing sanitization, lmk what you think